hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/ozoneimpl/ContainerReader.java

```diff
@@ -201,15 +202,26 @@ private void verifyContainerFile(long containerID,
    */
   public void verifyAndFixupContainerData(ContainerData containerData)
       throws IOException {
+    verifyAndFixupContainerData(containerData, false);
```

Comment:
Are these ContainerReader and KeyValueContainerUtil change related with this spread directory goal?  If not, could you put it into a new patch.